common/brave_switches.cc

```diff
@@ -23,4 +23,6 @@ const char kDisablePDFJSExtension[] = "disable-pdfjs-extension";
 // Allows disabling the Tor client updater extension.
 const char kDisableTorClientUpdaterExtension[] = "disable-tor-client-updater-extension";
 
+const char kUiMode[] = "ui-mode";
```

Comment:
nit: how about adding possible option values in the comment?

Reply:
good spot @simonhong - I added a comment now. Please could you re-review as it was automatically discarded.